ci-operator/config/openshift-kni/eapol-operator/openshift-kni-eapol-operator-main.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  authenticator-base:
+    name: "4.14"
+    namespace: ocp
+    tag: base-rhel9
+  builder:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.14
+  operator-base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  project_image:
+    dockerfile_path: images/build-root/Dockerfile
+images:
+- dockerfile_path: Dockerfile
+  from: operator-base
+  inputs:
+    builder:
+      as:
+      - golang:1.20
+  to: eapol-operator
+- dockerfile_path: Dockerfile.authenticator
+  from: authenticator-base
+  inputs:
+    builder:
+      as:
+      - docker.io/golang:1.20.2
+  to: eapol-authenticator
+promotion:
+  name: 0.0.1
+  namespace: ocp-kni
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: |
+    export XDG_DATA_HOME=${PWD}/bin
```

Comment:
might wanna document (in a comment) that you need a writeable location for the envtest